common/theme.c

```diff
@@ -486,4 +493,6 @@ void apply_theme_list_glyph(struct theme_config *theme, lv_obj_t *ui_lblItemGlyp
                                  LV_PART_MAIN | LV_STATE_FOCUSED);
     lv_obj_set_style_img_recolor_opa(ui_lblItemGlyph, theme->LIST_FOCUS.GLYPH_RECOLOUR_ALPHA,
                                      LV_PART_MAIN | LV_STATE_FOCUSED);
```

Comment:
I had taken this out and forgot to put this back in.  Default theme should have it's top and bottom padding set to zero